server/gitrest/packages/gitrest-base/src/utils/helpers.ts

```diff
@@ -44,6 +45,18 @@ export function validateBlobContent(content: string): boolean {
 	return content !== undefined && content !== null;
 }
 
+/**
+ * Returns the fsManagerFactory based on the isEphemeral flag
+ */
+export function getFilesystemManagerFactory(
```

Comment:
This is a bit confusing: RunnerFactory already has a function with the same name:

```ts
private getFileSystemManagerFactory(fileSystemName: string, config: Provider) {
		if (!fileSystemName || fileSystemName === "nodeFs") {
			return new NodeFsManagerFactory();
		} else if (fileSystemName === "redisFs") {
			return new RedisFsManagerFactory(config);
		}
		throw new Error("Invalid file system name.");
	}
```
Maybe we can rename these to avoid confusion?

Reply:
Makes sense. Renamed in RunnerFactory.